### PR TITLE
Add get method in collection and partition

### PR DESCRIPTION
**Pull request by fishpenguin**
Signed-off-by: fishpenguin <kun.yu@zilliz.com>

### CODE REVIEW

File: tests/test_partition.py [hunk 1]
```diff
@@ -74,3 +74,10 @@ def test_release(self, partition):
     def test_insert(self, partition):
         data = gen_list_data(default_nb)
         partition.insert(data)
+
+    @pytest.mark.xfail
+    def test_get(self, partition):
+        data = gen_list_data(default_nb)
+        ids = partition.insert(data)
+        assert len(ids) == default_nb
+        res = partition.get(ids[0:10])
```
Comment (scsven) [inline tests/test_partition.py:83]: Add an empty line at the end of file.

**Review by scsven — CHANGES_REQUESTED** [review]

**Review by scsven — APPROVED** [review]
/lgtm
/approve

**Review by czs007 — APPROVED** [review]

**Pull request merged**